src/components/TimeGridAllDay.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes, Component } from 'react'
+import message from '../utils/messages';
+
+import SelectableBackgroundCells from '../containers/SelectableBackgroundCells.jsx'
+
+export default class TimeGridAllDay extends Component {
+  static propTypes = {
+    messages: PropTypes.array,
+    gutterWidth: PropTypes.number,
+    range: PropTypes.array,
+    selectable: PropTypes.bool,
+    gutterRef: PropTypes.func
+  }
+  static defaultProps = {
+    range: [],
+    gutterRef: () => null
+  }
+  render() {
+    
+    return (
+      <div className="rbc-row">
+        <div ref={this.props.gutterRef} className="rbc-gutter-cell" style={this.props.gutterWidth ?
+          {width: this.props.gutterWidth} : {}}>
+          { message(this.props.messages).allDay }
+        </div>
+        <div className="rbc-allday-cell">
+          <SelectableBackgroundCells selectable={this.props.selectable}
```

Comment:
does the original code allow selection in this component? I don't actually think it makes sense to allow that but maybe I am disagreeing with an earlier me?